### DISCUSSION TITLE
建议适配： Bilibili 共享大会员 脚本

### DISCUSSION THREAD

**Discussion by mihomoQ** [the1812/Bilibili-Evolved#5061] [功能建议 / Ideas]
脚本链接：https://github.com/PC6live/bilibili-joybook-tampermonkey

脚本可与evolved正常共存 但在视频播放页面下载时无法检测到共享会员账号

**Comment by the1812** [comment]
![Snipaste_2024-12-04_23-34-21](https://github.com/user-attachments/assets/85924b89-3101-4c91-846e-9753e2c92cd9)
